src/gui/transferlistmodel.cpp

```diff
@@ -193,6 +193,7 @@ QVariant TransferListModel::headerData(const int section, const Qt::Orientation
             case TR_INFOHASH_V1: return tr("Info Hash v1", "i.e: torrent info hash v1");
             case TR_INFOHASH_V2: return tr("Info Hash v2", "i.e: torrent info hash v2");
             case TR_REANNOUNCE: return tr("Reannounce In", "Indicates the time until next trackers reannounce");
+            case TR_PRIVATE: return tr("Private", "Flags torrents form private trackers");
```

Comment:
Whether a torrent is private isn't defined by its source. Is disambiguation needed for this? I don't have any suggestions.

Reply:
How about simply putting "Flags private torrents"?

Reply:
Sure.

Reply:
Done